src/components/worker/index.ts

```diff
@@ -0,0 +1,20 @@
+// /* eslint-disable @typescript-eslint/ban-types */
+
+import {expose} from "threads/worker";
+import {getSSZType, createRandomValue} from "./helpers";
+
+const worker = {
+  createRandomValueFromSSZType(sszTypeName: string, forkName: string) {
+    const type = getSSZType(sszTypeName, forkName);
+    const value = createRandomValue(type);
+    return value;
+  },
+  serialize(sszTypeName: string, forkName: string, input: object) {
+    const type = getSSZType(sszTypeName, forkName);
+    const serialized = type.serialize(input);
+    const root = type.hashTreeRoot(input);
+    return {serialized, root};
+  }
+}
```

Comment:
we don't also deserialize?

Reply:
actually, i don't think we are currently.  i'm going to make an issue to investigate and add the worker properly for deserialization.